tf/modules/monitoring/variables.tf

```diff
@@ -1,36 +1,41 @@
 variable "cluster_name" {
-  type = string
+  type        = string
```

Comment:
This is my editor running `terraform fmt` on the source files, which I guess is ok.